### ISSUE TITLE
Unable to update node sdk to v12.4

### ISSUE THREAD

**Issue by nicksrandall** [DevCycleHQ/js-sdks#512]
You might have leaked a private NPM package or forgot to publish something?

```txt
This error happened while installing the dependencies of @devcycle/nodejs-server-sdk@1.12.4
@devcycle/bucketing-test-data is not in the npm registry, or you have no permission to fetch it.
```

**Comment by ajwootto** [comment]
hi @nicksrandall thanks for the report, it looks like something went wrong with our latest publish. Working on fixing it now. In the meantime any previously published version should work, such as [1.12.2](https://www.npmjs.com/package/@devcycle/nodejs-server-sdk/v/1.12.2)

**Comment by ajwootto** [comment]
@nicksrandall we've just published 1.12.5 which should fix this issue. Please let me know if you have any issues.
